src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Table/_shipping.html.twig

```diff
@@ -1,22 +1,17 @@
 {% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
 
 {% set orderShippingPromotionAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::ORDER_SHIPPING_PROMOTION_ADJUSTMENT') %}
+{% set shippingAdjustment = constant('Sylius\\Component\\Core\\Model\\AdjustmentInterface::SHIPPING_ADJUSTMENT') %}
 {% set orderShippingPromotions = sylius_aggregate_adjustments(order.getAdjustmentsRecursively(orderShippingPromotionAdjustment)) %}
 
 {% if not orderShippingPromotions is empty %}
     <td colspan="2" id="promotion-shipping-discounts">
-        <div class="ui relaxed divided list">
-            {% for label, amount in orderShippingPromotions %}
-                <div class="item">
-                    <div class="content">
-                        <div class="header">{{ label }}</div>
-                        <div class="description">
-                            {{ money.convertAndFormat(amount) }}
-                        </div>
-                    </div>
-                </div>
-            {% endfor %}
-        </div>
+        <div><strong>{{ 'sylius.ui.shipping'|trans }}:</strong></div>
+
+        {% for label, amount in orderShippingPromotions %}
```

Comment:
Can we verify the behavior for two shipping promotions at the same time? 

I'm not sure, but it seems, that we should show something like this instead:

```suggestion
            <span class="old-price">{{ money.convertAndFormat(order.getAdjustmentsTotal(shippingAdjustment)) }}</span>
            {{ money.format(order.getAdjustmentsTotal(shippingAdjustment) + order.getAdjustmentsTotal(orderShippingPromotionAdjustment), order.currencyCode) }}
```

We want to show the aggregated shipping price reduced by aggregated shipping promotion. In the current scenario, we will show two different shipping prices (if i'm not wrong :P)